### PR TITLE
feat : added log in page

### PR DESCRIPTION
**Pull request by krate27**
Screenshot (552)

### CODE REVIEW

File: global.css [hunk 1]
```diff
@@ -33,4 +34,7 @@ h2{
 
 h3{
```
Comment (raghav6202) [inline global.css:35]: Try to use class selectors as much as possible

**Review by raghav6202 — APPROVED** [review]
Looks good to merge. Nice U.I 👍

File: global.css (hunk 1 above)
Comment (krate27) [reply global.css:35]: yes agreed, but this was special case. Thank you for reviewing Raghav

**Comment by krate27** [comment]
> Looks good to merge. Nice U.I 👍

Thank you Raghav

**Review by rahul0396 — COMMENTED** [review]
Do changes as raghav suggest for using class selector and the remaining code is good. Good to merge .

**Pull request merged**